cli/command/engine/client_test.go

```diff
@@ -69,9 +69,9 @@ func (w *fakeContainerizedEngineClient) DoUpdate(ctx context.Context,
 	opts clitypes.EngineInitOptions,
 	out clitypes.OutStream,
 	authConfig *types.AuthConfig,
-	healthfn func(context.Context) error) error {
+	healthfn func(context.Context) error, runtimeMetadataDir string) error {
```

Comment:
Same here; Instead of adding another argument, can we add `runtimeMetadataDir` as a field on `clitypes.EngineInitOptions` ?

Reply:
Wait: what's that `healthfn` argument actually for? Looks like it's only used for testing? That really shouldn't be in there; I'm sure that can be set on the mock during testing?

Reply:
Hmm.. hold over from prior implementations... we really need to get this in ASAP so let me excise that part in a follow up if you don't mind.